File: Julia/filenamemake.jl
#Function for constructing filenames
#23/07/2017

function filenamemake(atom,nn,ll,jj)

  #Avoid extra "." in filename
  jstring = replace(string(jj),".","_")

  filename = string("WaveFunctions/",atom,"n",string(nn),"l",string(ll),"j",jstring,".jld")

end

File: Matlab/filenamemake.m
%Function for constructing filenames
%24/07/2017

function filename = filenamemake(atom,nn,ll,jj)

%Avoid extra '.' in filename
jstring = strrep(num2str(jj),'.','_');

filename = ['Wavefunctions\',...
    atom,num2str(nn),'n',num2str(ll),'l',jstring,'j.mat'];